src/services/require-install.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path');
+const semver = require('semver');
+const Plugins = require('@oclif/plugin-plugins').default;
+const corePJSON = require('../../package.json');
+const { logger } = require('./messaging/logging');
+
+/**
+ * Retrieves the plugin for a given command.
```

Comment:
Do we need a placeholder here to refactor for when https://github.com/oclif/config/issues/91 gets resolved?

Reply:
I'm not sure if it will fix the issue of needing to determine the _top-level_ plugin.